addons/edi/models/edi_connection_mail.py

```diff
@@ -76,6 +76,8 @@ def send_outputs(self, _conn, path, _transfer):
             vals['res_id'] = path.id
             vals['email_to'] = path.path
             vals['attachment_ids'] = [(6, 0, [x.id for x in attachments])]
+            vals['mail_server_id'] = _transfer.gateway_id.outgoing_server_id
```

Comment:
* This should be conditional on that value
* _transfer should be renamed to transfer if it is now in use

Reply:
Yeah realised this, only if its set it gets updated now